src/other/PropertyEditor.js

```diff
@@ -74,6 +79,20 @@
         ;
         table
             .each(function (d) {
+                if (d instanceof PropertyEditor === false) {
+                    if (context.overrideTags() && Object.keys(context.overrideTags()).length) {
```

Comment:
@jbrundage should this only work then on initial render? ... when i was testing and all it was nice to be able to update this stuff on the fly ...